donate/templates/pages/core/contributor_support_page_master.html

```diff
@@ -7,18 +7,24 @@
 {% endblock %}
 
 {% block head_addenda %}
-    <script src="https://www.google.com/recaptcha/api.js"></script>
-    <script>
-        function timestamp() {
-            var response = document.getElementById("g-recaptcha-response");
-            if (response == null || response.value.trim() == "") {
-                var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);
-                elems["ts"] = JSON.stringify(new Date().getTime());
-                document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems);
-            }
+    <script type="text/javascript">
+    function hookupFormSubmitButton() {
+        try {
+            const recaptcha = document.getElementById("g-recaptcha");
+            console.log(recaptcha);
+            grecaptcha.render(recaptcha, {
+                sitekey: "{{ help_recaptcha_site_key }}",
+                callback: (token) => {
+                    var submitButton = document.querySelector('.support-form input[type="submit"]');
+                    submitButton.removeAttribute("disabled");
```

Comment:
note that we're not _verifying_ this recaptcha token atm, we're simply using it to unlock the submit button. That's not "secure" (an issue has been filed to determine whether we need to do server validation before proxying the data on to our help endpoint).